tests/controllers/TalksControllerTest.php

```diff
@@ -33,17 +44,23 @@ public function testClaimTalkWithNoUserIdThrowsException()
      *
      * @return void
      *
-     * @test
```

Comment:
Removed a few of these too - all of the functions are prefixed with test so seems pointless to have the test annotation too